### PR TITLE
Add Kafka Manager chart

### PR DESCRIPTION
**Pull request by oleh-ozimok**

### CODE REVIEW

File: stable/kafka-manager/values.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+replicaCount: 1
+image:
+  repository: sheepkiller/kafka-manager
+  tag: alpine
```
Comment (unguiculus) [inline stable/kafka-manager/values.yaml:4]: Can you use a fix release version?

File: stable/kafka-manager/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## TL;DR;
+
+```bash
+$ helm install stable/kafka-manager
+```
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```
Comment (unguiculus) [inline stable/kafka-manager/README.md:13]: 1.8+

File: stable/kafka-manager/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1beta1
```
Comment (unguiculus) [inline stable/kafka-manager/templates/deployment.yaml:1]: We offically support the current and previous minor versions of Kubernetes. Currently that would be 1.8+. You should use `apps/v1beta2`.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. I have just a few comments.

**Comment by tedmiston** [comment]
It looks like this PR is a bit stale.  I've pulled in the branch, and made the changes specified in comments and am able to run it myself locally.  Would it be worthwhile to open a fresh PR to push this over the finish line?

I looked into the above failed test `pull-charts-e2e`, but am unclear on how to resolve the error:

```
I0302 18:07:44.237] [info] o.a.z.ZooKeeper - Client environment:os.arch=amd64
I0302 18:07:44.238] [info] o.a.z.ZooKeeper - Client environment:os.version=4.4.86+
I0302 18:07:44.238] [info] o.a.z.ZooKeeper - Client environment:user.name=root
I0302 18:07:44.238] [info] o.a.z.ZooKeeper - Client environment:user.home=/root
I0302 18:07:44.238] [info] o.a.z.ZooKeeper - Client environment:user.dir=/kafka-manager-1.3.1.6
I0302 18:07:44.239] [info] o.a.z.ZooKeeper - Initiating client connection, connectString= sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@20fd4379
I0302 18:07:44.239] [info] o.a.z.ClientCnxn - Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
I0302 18:07:44.239] [warn] o.a.z.ClientCnxn - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
I0302 18:07:44.240] java.net.ConnectException: Connection refused
I0302 18:07:44.240] 	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[na:1.8.0_92]
I0302 18:07:44.240] 	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717) ~[na:1.8.0_92]
I0302 18:07:44.240] 	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361) ~[org.apache.zookeeper.zookeeper-3.4.6.jar:3.4.6-1569965]
I0302 18:07:44.241] 	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081) ~[org.apache.zookeeper.zookeeper-3.4.6.jar:3.4.6-1569965]
I0302 18:07:44.241] [info] k.m.a.KafkaManagerActor - zk=
I0302 18:07:44.241] [info] k.m.a.KafkaManagerActor - baseZkPath=/kafka-manager
I0302 18:07:44.241] [info] play.api.Play - Application started (Prod)
I0302 18:07:44.242] [info] p.c.s.NettyServer - Listening for HTTP on /0:0:0:0:0:0:0:0:9000
```

Anyone have advice for this?  Is it possibly something specific to the CI environment?

**Comment by oleh-ozimok** [comment]
Was busy, sorry. @tedmiston kafka-manager need zookeper server(s) for start. I can add zookeer as  dependency, but I do not think that this is a good idea.

**Comment by tedmiston** [comment]
Ah I must have botched the config passing them through.  I'll take another go at this today.  Thank you.

**Comment by sneko** [comment]
Any news about this chart?

**Comment by tedmiston** [comment]
I’m no longer using Kafka / Kafka Manager personally but please tag me if there’s anything I can do to help be useful here.

**Comment by unguiculus** [comment]
Closing in favor of #8715

**Pull request closed without merging**